src/java/org/apache/nutch/crawl/CrawlDb.java

```diff
@@ -236,10 +237,10 @@ public int run(String[] args) throws Exception {
    * Used for Nutch REST service
    */
   @Override
-  public Map<String, Object> run(Map<String, String> args, String crawlId) throws Exception {
+  public Map<String, Object> run(Map<String, Object> args, String crawlId) throws Exception {
```

Comment:
@lewismc, making it object allows me to parse multiple inputs in the Map args(for ex- segments in the updatedb job) as an Arraylist instead of string parsing. Also this change the 1x code similar to 2x and also makes porting the webui easier as it expects an Object.